tools/pipelines/templates/build-npm-client-package.yml

```diff
@@ -569,13 +575,22 @@ extends:
               # Note that we need to disable it in the right way for 1ES pipeline templates, vs manual CodeQL tasks.
               - name: ONEES_ENFORCED_CODEQL_ENABLED
                 value: 'false'
+              - name: COMMIT_SHA
+                value: $[ dependencies.build.outputs['setCommitSHA.COMMIT_SHA'] ]
+
             steps:
               # Setup
               - checkout: self
                 clean: true
                 lfs: '${{ parameters.checkoutSubmodules }}'
                 submodules: '${{ parameters.checkoutSubmodules }}'
 
+              - script: |
```

Comment:
There is some pending changes here to unify repeated steps through tests jobs. I'll work on that on a separate PR and after merging some other issues in the pipeline.